docs/userguide/pyproject_config.rst

```diff
@@ -63,6 +64,7 @@ The ``project`` table contains metadata fields as described by
    [project.scripts]
    my-script = "my_package.module:function"
 
+   # ... other project metadata fields as specified in PEP 621
```

Comment:
The `pyproject.toml` standard is a live standard, so I think it is better if we add the up-to-date field.

```suggestion
   # ... other project metadata fields as specified in:
   #     https://packaging.python.org/en/latest/specifications/declaring-project-metadata/
```